### ISSUE TITLE
Any paper / article describing procedure is availible?

### ISSUE THREAD

**Issue by mrgloom** [ZFTurbo/Keras-inference-time-optimizer#17]
Any paper / article describing procedure is availible? How it's officially called, batch norm fusion?

https://tkv.io/posts/fusing-batchnorm-and-conv/

**Comment by ZFTurbo** [comment]
Yes, it's batchnorm fusion.
